src/external/rawspeed/data/cameras.xml

```diff
@@ -5585,6 +5585,17 @@
 		<Crop x="4" y="0" width="-36" height="0"/>
 		<Sensor black="1" white="15868"/>
 	</Camera>
+        <Camera make="RICOH IMAGING COMPANY, LTD." model="PENTAX K-3 II">
+                <ID make="Pentax" model="K-3 II">PENTAX K-3 II</ID>
+                <CFA width="2" height="2">
+                        <Color x="0" y="0">RED</Color>
+                        <Color x="1" y="0">GREEN</Color>
+                        <Color x="0" y="1">GREEN</Color>
+                        <Color x="1" y="1">BLUE</Color>
+                </CFA>
+                <Crop x="4" y="0" width="-36" height="0"/>
+                <Sensor black="1" white="15865"/>
```

Comment:
iminimal: black/white is 1/15865 for iso 800, 1/15866 (iso 1600),  2/15865 (iso 3200)